### ISSUE TITLE
Accessibility: controls within calls and first two controls on profile screen are not labelled

### ISSUE THREAD

**Issue by ethindp** [signalapp/Signal-Android#11658]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Using TalkBack, certain controls within the Signal user interface are not labelled properly. Though I could create my own labels, these are only temporary and will need to be recreated whenever the UI changes. In particular, the following controls (that I've found anyway) aren't labelled:

* Any of the controls that are shown while in a call
* The two unlabelled controls at the top of the contact info screen

### Steps to reproduce
- Enable TalkBack and log into signal
- Start a call
- Navigate the screen (swiping left/right) and attempt to determine what each control does

Or, for the second set of controls:

- Log into signal and enable TalkBack
- Open a conversation with a contact
- Tap on the control that starts with "Contact photo image"
- Navigate the screen; the first two controls (swiping right) are unlabelled

**Actual result:** The controls within calls, and the first two controls in a contacts profile screen, are unlabelled.
**Expected result:** I should be able to navigate the call controls and contact profile screen and know what each control does.

### Device info

**Device:** Pixel 3A
**Android version:** 11
**Signal version:** 5.24.7

### Link to debug log

https://debuglogs.org/e860eb9e428aa35256a7fdf91f96ce760711f4c8a6a2fa77c4e23fb385defdf7

**Comment by ethindp** [comment]
This is what I'm wondering. The issue still exists.

**Comment by alex-signal** [comment]
Definitely still something we want to do. There are a lot of issues with Talkback.

**Comment by davidbtc2009** [comment]
This is also related to #9774 which was closed with no resolution

**Comment by clado** [comment]
Hi everyone! As of #11977 and #11996 buttons on the call screen should be labelled. These fixes should both be included in version 5.35.0 or higher.

I know that this issue also references two unlabelled buttons on the contact profile screen, and for that reason it has not yet been fully resolved. I have a commit started to fix those two controls, but haven't pushed it yet because I've only fixed one of them so far. ^^;

While testing the in-call buttons I noticed that the experience of using them may not be ideal.
1. It looks like users may need to tap on something unlabelled in order to access the buttons?
2. The buttons seemed to disappear pretty quickly while interacting with them, making them difficult to use imo?

@davidbtc2009 @ethindp I think you both know a lot more about TalkBack and UX than me. Would you be willing to investigate the new in-call labels and create some new GitHub issues if you can confirm the UX concerns I mentioned or if you find any additional ones?

I have pretty limited time/energy, but am really interested in using some of it to (slowly) work on TalkBack bugs, so please feel free to tag me in anything you create! :)

**Comment by ethindp** [comment]
I've been on a signal call recently and things were labelled as expected. I would encourage this issue to be relabeled as not "wontfix", since that labeling was (I'm pretty sure) automatically set. I would've fixed this problem long ago but I do not have a development environment for Android set up and getting one set up is impossible (Java apps are not accessible on Linux, blame Oracle). Why Google chose Java of all languages to write an IDE in I will never understand. Hell, why the makers of Android chose Java at all is a mystery. Anyway, things to be labelled fine now from what I can tell, though I'm not using Talkback so...
